recipes/gemfile.rb

```diff
@@ -13,25 +13,19 @@
     gem 'strong_migrations'
   HEREDOC
 end
-append_to_file 'Gemfile' do
+append_to_file 'Gemfile', after: /group :development do\n/ do
```

Comment:
Does this work for you? I've tried it on Ruby 3.4.1 with Rails 8.0.2 and it's not working because new app doesn't have a "development-only" bundler group.